CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ Updates should follow the [Keep a CHANGELOG](https://keepachangelog.com/) princi
 
 ## [Unreleased][unreleased]
 
+## [0.1.5] - 2021-09-20
```

Comment:
The release date needs to be changed.

Or, if you aren't quite ready to release this immediately, you should temporarily remove this line plus the changes at the bottom of this until you're just about to make that release.

Reply:
Noted! This release will go out tomorrow.